### DISCUSSION TITLE
Connecting the inverter

### DISCUSSION THREAD

**Discussion by videopix** [hoylabs/OpenDTU-OnBattery#124] [General]
Hi,
I suppose the PV setup for your project is similar to what's shown in https://solarmodule-gladbeck.de/600w-balkonkraftwerk-mit-speicher-und-hoymiles-100w-dauereinspeisung.
The Hoymiles inverter for a 600 watt system has two pairs of MC4 connectors for two panels. On the schematic, however, it is only connected to the battery with one pair. I'm assuming that the second pair will remain unconnected. Looks like this can never feed the allowed 600 watts. With the max allowed current of 11.5A and 24 volts it would only reach 276 watts.
Or did I get something wrong?

**Comment by helgeerbe** [comment]
Hi @videopix I have an similar setup. For my battery PV I use a HM-300 with a single string. So I draw a max power of 300W from the battery. If your battery is capable to provide 600W and you want to make use of it. you could attach both strings to the battery in parallel. For the battery it's just a second consumer.

By the way, in the link you provide a MPPT 100/15 is shown. This seems to be a little small for me. Victron has a calculator for the MPPTs because the sizing depends of the panels you use.

**Reply by videopix** [reply]
Hi Helge,
Thanks for your answer and for pointing me to the Victron MPPT calculator!
Could'nt connecting several strings to the battery in parrallel potentially lead to interferences of the HM-internal MPPTs? I read that they are not active at relatively low voltages. But when I would upgrade to a 48V battery system to be able to feed more power when needed, do you think this might cause problems?

**Reply by helgeerbe** [reply]
I just. use a battery with 24 V. So I don't now what issues you might have with 48V. @berni2288 does your battery operates with 48V?

**Reply by berni2288** [reply]
I run a Pylontech 48V battery, connected in parallel to both (2 pos + 2 neg) DC inputs of the HM-800, and it works just fine. There are no signs of any weird behavior.
But the MPPT range of my HM-800 is 34-48V, so I normally don't fall into this range because my battery voltage is usually higher than the MPPT range.
So above 48V everything is really stable and that was actually the reason why I bought a 48V battery.
I don't know how the MPPT tracker would behave when 2x 24V inputs are connected.
